srcpkgs/riot-desktop/INSTALL.msg

```diff
@@ -1,10 +1,10 @@
-Since Riot 1.6, it uses Electron 8. This version of Electron has
-dropped support for the older XEmbed system tray API. If you are using
-a desktop or window manager that does not support StatusNotifierItems
-(i3, awesome, stalonetray, etc), you should install and configure
-snixembed:
+Riot 1.6.8 has upgraded to Electron 9, with support for tray icons. In
+most cases, it is no longer necessary to use a package like snixembed.
+
+If using an older version of Riot 1.6, and a desktop or window manager
```

Comment:
Why mention older versions? That install.msg will only ever be seen from users with Riot 1.6.8. Also imo at that point you could just remove it completely.

Reply:
Changed it around a bit. I think it's good to keep it around for a while, at least, for the people who installed snixembed.

Reply:
Is having snixembed make bad in any way?

Reply:
It's not bad, but it was created to solve a specific problem, which was Electron dropping support for XEmbed. That support has returned, so snixembed is thus no longer necessary for Riot to have a tray icon on window managers that don't support StatusNotifierItems.

Reply:
@Johnnynator @ericonr can we consider this comment resolved?

Reply:
For me it's fine, but @Johnnynator has the final word.